tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -255,3 +258,63 @@ def test_positive_prism_central_prism_central_option(
         command = get_configure_command(virtwho_config['id'], default_org.name)
         deploy_configure_by_command(command, form_data['hypervisor-type'], org=default_org.label)
         assert get_configure_option("prism_central", config_file) == 'true'
+
+    @pytest.mark.tier2
+    def test_positive_ahv_internal_debug_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify ahv_internal_debug option by hammer virt-who-config"
+
+        :id: a169a843-6ec7-4ef1-b471-649304e77cf1268
+
+        :expectedresults:
+            enable-ahv-debug option has been set to no
+            ahv_internal_debug bas been set to false in virt-who-config-X.conf
+            warning message exist in log file /var/log/rhsm/rhsm.log
+            ahv_internal_debug option can be updated
+            message Host UUID {system_uuid} found for VM: {guest_uuid} exist in rhsm.log
+            ahv_internal_debug bas been set to true in virt-who-config-X.conf
+            warning message does not exist in log file /var/log/rhsm/rhsm.log
```

Comment:
same numbering the steps can be done as above.

Reply:
> same numbering the steps can be done as above.

@shubhamsg199 Have resolved